runtime/lua/vim/lsp/client.lua

```diff
@@ -122,6 +149,12 @@ local validate = vim.validate
 --- Useful for buffer-local setup.
 --- @field on_attach fun(bufnr: integer)
 ---
+--- @field private _before_init_cb? vim.lsp.client.before_init_cb
+--- @field private _on_attach_cbs vim.lsp.client.on_attach_cb[]
+--- @field private _on_init_cbs vim.lsp.client.on_init_cb[]
```

Comment:
Could we use `package` instead of `private` here to avoid the `diagnostic invisible` annotations?

Reply:
Strangely `package` has different semantics on non-methods and you still get diagnostics.